<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border: none;
	background-color: transparent;
	background: none;
	padding: 0;
	margin: 0;
}
#centralwidget{
	background-color: #1f232a;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QGridLayout" name="gridLayout" columnstretch="0">
          <item row="1" column="0">
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,5">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QStackedWidget" name="stackedWidget">
               <widget class="QWidget" name="page"/>
               <widget class="QWidget" name="page_2"/>
              </widget>
             </item>
             <item>
              <widget class="QStackedWidget" name="graph_stack">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>400</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #d6d6d6;
border-radius: 20px;</string>
               </property>
               <widget class="QWidget" name="stackedWidget_7Page1">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="controls_widget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>2</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>2</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="status_label">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #d6d6d6;
border-radius: 8px;
color: #1d1d1b;
</string>
            </property>
            <property name="text">
             <string>Predicted Output: </string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="status_widget" native="true">
         <property name="styleSheet">
          <string notr="true">	color: #fff;</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="1,0">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <property name="horizontalSpacing">
              <number>35</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">	color: #fff;</string>
               </property>
               <property name="text">
                <string>Controls</string>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::LinksAccessibleByMouse</set>
               </property>
              </widget>
             </item>
             <item row="12" column="0">
              <widget class="QPushButton" name="pushButton_2">
               <property name="styleSheet">
                <string notr="true">QPushButton{
color: #fff;
background-color: rgb(152, 152, 152);
border: 2px solid rgb(83, 83, 83);
border-radius: 12px;
}
QPushButton:hover{
	border: 2px solid rgb(48, 50, 62);
	
	background-color: rgb(86, 86, 86);
}
QPushButton:focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
               </property>
               <property name="text">
                <string>Analyze</string>
               </property>
              </widget>
             </item>
             <item row="13" column="0">
              <widget class="QPushButton" name="pushButton">
               <property name="styleSheet">
                <string notr="true">QPushButton{
color: #fff;
background-color: rgb(152, 152, 152);
border: 2px solid rgb(83, 83, 83);
border-radius: 12px;
}
QPushButton:hover{
	border: 2px solid rgb(48, 50, 62);
	
	background-color: rgb(86, 86, 86);
}
QPushButton:focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
               </property>
               <property name="text">
                <string>Play</string>
               </property>
              </widget>
             </item>
             <item row="12" column="1">
              <widget class="QPushButton" name="pushButton_6">
               <property name="styleSheet">
                <string notr="true">QPushButton{
color: #fff;
background-color: rgb(152, 152, 152);
border: 2px solid rgb(83, 83, 83);
border-radius: 12px;
}
QPushButton:hover{
	border: 2px solid rgb(48, 50, 62);
	
	background-color: rgb(86, 86, 86);
}
QPushButton:focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
               </property>
               <property name="text">
                <string>Process</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QPushButton" name="pushButton_5">
               <property name="styleSheet">
                <string notr="true">QPushButton{
color: #fff;
background-color: rgb(152, 152, 152);
border: 2px solid rgb(83, 83, 83);
border-radius: 12px;
}
QPushButton:hover{
	border: 2px solid rgb(48, 50, 62);
	
	background-color: rgb(86, 86, 86);
}
QPushButton:focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
               </property>
               <property name="text">
                <string>Opne Audio</string>
               </property>
              </widget>
             </item>
             <item row="14" column="1">
              <widget class="QPushButton" name="pushButton_7">
               <property name="styleSheet">
                <string notr="true">QPushButton{
color: #fff;
background-color: rgb(152, 152, 152);
border: 2px solid rgb(83, 83, 83);
border-radius: 12px;
}
QPushButton:hover{
	border: 2px solid rgb(48, 50, 62);
	
	background-color: rgb(86, 86, 86);
}
QPushButton:focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
               </property>
               <property name="text">
                <string>Show MEL</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QLineEdit" name="lineEdit_2">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
	border: 2px solid rgb(83, 83, 83);
	border-radius: 13px;
	color: #fff;
	padding-left: 15px;
	padding-right: 15px;
	background-color: rgb(34, 36, 44);
}
QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
}
</string>
               </property>
               <property name="text">
                <string>Filter Cutoff: 5HZ</string>
               </property>
              </widget>
             </item>
             <item row="13" column="1">
              <widget class="QPushButton" name="pushButton_8">
               <property name="styleSheet">
                <string notr="true">QPushButton{
color: #fff;
background-color: rgb(152, 152, 152);
border: 2px solid rgb(83, 83, 83);
border-radius: 12px;
}
QPushButton:hover{
	border: 2px solid rgb(48, 50, 62);
	
	background-color: rgb(86, 86, 86);
}
QPushButton:focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
               </property>
               <property name="text">
                <string>Show FFT</string>
               </property>
              </widget>
             </item>
             <item row="14" column="0">
              <widget class="QPushButton" name="pushButton_4">
               <property name="styleSheet">
                <string notr="true">QPushButton{
color: #fff;
background-color: rgb(152, 152, 152);
border: 2px solid rgb(83, 83, 83);
border-radius: 12px;
}
QPushButton:hover{
	border: 2px solid rgb(48, 50, 62);
	
	background-color: rgb(86, 86, 86);
}
QPushButton:focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
               </property>
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
             <item row="15" column="1">
              <widget class="QPushButton" name="pushButton_9">
               <property name="styleSheet">
                <string notr="true">QPushButton{
color: #fff;
background-color: rgb(152, 152, 152);
border: 2px solid rgb(83, 83, 83);
border-radius: 12px;
}
QPushButton:hover{
	border: 2px solid rgb(48, 50, 62);
	
	background-color: rgb(86, 86, 86);
}
QPushButton:focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
               </property>
               <property name="text">
                <string>Show Spectrogram</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">	color: #fff;</string>
               </property>
               <property name="text">
                <string>Input Parameters</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QSlider" name="horizontalSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Resource.qrc"/>
  <include location="../../sample Feature/Res.qrc"/>
 </resources>
 <connections/>
</ui>
